automation/docker/application.conf.ctmpl

```diff
@@ -27,6 +27,9 @@ fireCloud {
   thurloeApiUrl = "https://thurloe-fiab.dsde-{{$environment}}.broadinstitute.org:25443/"
   {{end}}
   fireCloudId = "{{$secrets.Data.firecloud_id}}"
+
+  # TODO more Config class nonsense
```

Comment:
We're still iterating on automation-test configs in workbench-libs.  This will be part of https://broadinstitute.atlassian.net/browse/GAWB-3110